### PR TITLE
display content first and then load menus

### PR DESCRIPTION
**Pull request by max-nextcloud**
Rendering the menubar and menububble takes a significant amount of time.
First display the content and then render the menus.

* Resolves: #2183 
* Target version: master 

Renders the content >500ms faster on my local dev setup.

Now the content is drawn (LCP) before the large computation load of rendering the menubar:
Bildschirmfoto von 2022-03-07 10-19-02

### CODE REVIEW

**Comment by max-nextcloud** [comment]
This should also speed up opening files but the content seems to render at the same time as the menubar there. Will need to investigate if it actually speeds up loading there as well or what is blocking it.

**Comment by vinicius73** [comment]
We also can use [window.requestAnimationFrame()](https://developer.mozilla.org/pt-BR/docs/Web/API/Window/requestAnimationFrame) to "delay" some stuff to create a more smooth experience.

File: src/components/EditorWrapper.vue [hunk 1]
```diff
@@ -34,8 +34,8 @@
 			</p>
 		</div>
 		<div v-if="displayed" id="editor-wrapper" :class="{'has-conflicts': hasSyncCollission, 'icon-loading': !initialLoading && !hasConnectionIssue, 'richEditor': isRichEditor, 'show-color-annotations': showAuthorAnnotations}">
-			<div id="editor">
-				<MenuBar v-if="tiptap && !syncError && !readOnly"
+			<div v-if="tiptap" id="editor">
+				<MenuBar v-if="initialLoading && !syncError && !readOnly"
```
Comment (vinicius73) [inline src/components/EditorWrapper.vue:38]: Should it be better to use a computed here?

**Comment by max-nextcloud** [comment]
> We also can use [window.requestAnimationFrame()](https://developer.mozilla.org/pt-BR/docs/Web/API/Window/requestAnimationFrame) to "delay" some stuff to create a more smooth experience.

For now i am mostly trying to render the content as early as possible. In the rich workspaces in files the menubar will not show anyway. So we can easily speed up the loading of the text by postponing the menubar loading.
When opening a file to edit it would be nice if the menubar was snappy... however the content still seems more important.

I did read upon `requestAnimationFrame` but i am not sure how it would apply here. What's the effect you would want to achieve?

**Comment by vinicius73** [comment]
> I did read upon `requestAnimationFrame` but i am not sure how it would apply here. What's the effect you would want to achieve?

Nothing for now @max-nextcloud, maybe in new interactions about performance improvements.

**Review by vinicius73 — APPROVED** [review]

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**